Makefile

```diff
@@ -34,6 +34,7 @@ all:
 
 bootstrap:
 	${DEV_ENV_CMD} glide install
+	go get github.com/pwaller/goupx
```

Comment:
@aledbf is `goupx` available in a docker image somewhere? We'd like to keep the build tools encapsulated inside Docker images as much as possible.